README.md

```diff
@@ -462,47 +462,48 @@ interface ScrollyBlock extends Parent {
 ```ts
 interface ScrollySection extends Parent {
 	type: "scrolly-section"
-	theme: "dark-text" | "light-text" | "dark-text-no-box" | "light-text-no-box"
+	display: "dark-background" | "light-background"
+	noBox?: true,
 	position: "left" | "center" | "right"
 	transition?: "delay-before" | "delay-after"
-	children: [ImageSet, ...ScrollyCopy[]]
+	children: [ScrollyImage, ...ScrollyCopy[]]
 }
 ```
 
 **ScrollySection** represents a section of a [ScrollyBlock](#scrollyblock)
 
 - TODO: could `transition` have a `"none"` value so it isn't optional?
 
+### `ScrollyImage`
+
+```ts
+interface ScrollyImage extends Node {
+	type: "scrolly-image"
+	id: string
+}
+```
+
+**ScrollyImage** represents an image contained in a [ScrollySection](#scrollysection)
+
 ### `ScrollyCopy`
 
 ```ts
 interface ScrollyCopy extends Parent {
 	type: "scrolly-copy"
-	children: ScrollyText[]
+	children: (ScrollyHeading | Paragraph)[]
 }
 ```
 
 TODO is this badly named?
 
-**ScrollyCopy** represents a collection of **ScrollyText** nodes.
+**ScrollyCopy** represents a collection of **ScrollyHeading** or **Paragraph** nodes.
 
 ```ts
-interface ScrollyText extends Parent {
-	type: "scrolly-text"
-	level: string
-}
-
-interface ScrollyHeading extends ScrollyText {
-	type: "scrolly-text"
+interface ScrollyHeading extends Parent {
+	type: "scrolly-heading"
 	level: "chapter" | "heading" | "subheading"
 	children: Text[]
 }
-
-interface ScrollyParagraph extends ScrollyText {
-	type: "scrolly-text"
-	level: "text"
-	children: Phrasing[]
-}
 ```
 
 **ScrollyText** represents an individual unit of copy for a
```

Comment:
**issue**: this description should be updated to describe `ScrollyHeading` instead